index.test.js

```diff
@@ -43,6 +43,7 @@ describe('Deploy to ECS', () => {
 
         core.getInput = jest
             .fn()
+            .mockReturnValueOnce('1') // desired-count
```

Comment:
Instead of updating the existing tests (which should work the same), can we add a test for the positive case, where `desiredCount` is explicitly set?